implementation/src/main/java/io/smallrye/mutiny/Uni.java

```diff
@@ -793,4 +793,15 @@ static UniCombine combine() {
     default <O> Uni<O> replaceWith(Uni<O> uni) {
         return onItem().transformToUni(ignore -> uni);
     }
+
+    /**
+     * Ignore the item emitted by this {@link Uni} and replace it with {@code null}.
+     * <p>
+     * This is a shortcut for {@code uni.onItem().transform(ignore -> null)}.
+     * 
+     * @return the new {@link Uni}
+     */
+    default Uni<T> replaceWithNull() {
+        return onItem().transform(ignore -> null);
```

Comment:
Do we want the same method in UniOnItem?
As you added in UniOnFailure, it would make sense.

Reply:
This could be added as part of the thought process in #319 (/cc @gavinking)

In `UniOnItem` right now we also have `uni.onItem().ignore().andContinueWithNull()`